packit_service/service/events.py

```diff
@@ -786,6 +889,25 @@ def get_copr_build_logs_url(event: CoprBuildEvent) -> str:
     )
 
 
+def get_koji_build_logs_url(event: KojiBuildEvent) -> Optional[str]:
+    if not event.rpm_build_task_id:
+        return None
+
+    return (
+        f"https://kojipkgs.fedoraproject.org//work/tasks/"
```

Comment:
we should soon prioritize the distinction b/w koji instances so this would work for centos as well